packages/chart/src/components/LineChart/helpers.ts

```diff
@@ -1,31 +1,64 @@
 import { DataItem, StyleProps, Style } from './LineChartProps'
 import { PreliminaryDataItem } from '../../types/ChartConfig'
-import { getTextWidth } from '@cdc/core/helpers/getTextWidth'
+
 import _ from 'lodash'
 export const createStyles = (props: StyleProps): Style[] => {
-  const { preliminaryData, data, stroke, strokeWidth, handleLineType, lineType, seriesKey } = props
+  const {
+    preliminaryData,
+    data,
+    stroke,
+    strokeWidth,
+    handleLineType,
+    lineType,
+    seriesKey,
+    dynamicCategory,
+    originalSeriesKey
+  } = props
+  const dynamicData = data.filter(d => {
```

Comment:
Add checks for data before filtering.
Add checks for preliminaryData before running forEach
Add checks for dynamicCategory before attemptying to acces d[dynamicCategory]

Reply:
Fixed!